### PR TITLE
{toolchain} intel/2019.00

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/24493c2f15edeb1dafa782337f9ae09c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e07575cb75a3b72a09f4aa355278bab8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/bd02a14b795b4d1aa068ae8b36b500a3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/b17994e6c56bb2cbba0f3785c732d732 for a full test report.

File: easybuild/easyconfigs/i/ifort/ifort-2019.0.117-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -8,6 +8,7 @@ description = "Intel Fortran compiler"
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/14866/']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/i/ifort/ifort-2019.0.117-GCC-8.2.0-2.31.1.eb]: `s/14866/13583`

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/f08c3632133d3c33e8c27dd1f043db87 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**